docs/src/gcode/o-code_fr.adoc

```diff
@@ -225,13 +371,12 @@ O[#101+2] call
 .Calcul des valeurs dans les O-codes
 Voici un condensé des sections utiles aux calculs des O-codes:
 
-* <<sec:parametres, les paramètres>>, 
-* <<sec:Expressions, les expressions>>, 
-* <<sec:Operateurs-Binaires, les opérateurs binaires>>, 
-* <<sec:Fonctions, les fonctions>>.
+* <<sec:parametres,les paramètres>>,
+* <<sec:Expressions,les expressions>>,
+* <<sec:Operateurs-Binaires,les opérateurs binaires>>,
+* <<sec:Fonctions,les fonctions>>.
```

Comment:
```suggestion
* <<sec:ocodes-parametres,Les paramètres>>,
* <<sec:ocodes-expressions,Les expessions>>,
* <<sec:ocodes-operateurs-binaires,Les opérateurs binaires>>,
* <<sec:ocodes-fonctions,Les fonctions>>.
```

Reply:
In my understanding, references need to be fixed in the English original.